### PR TITLE
Translate several version notes into Simplified Chinese (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

**Comment by hlcfan** [comment]
Looking good, thanks!

File: zh_cn/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* Ruby 2.3 系列： 2.3.5 及之前版本
+* Ruby 2.4 系列： 2.4.2 及之前版本
+* Ruby 2.5 系列： 2.5.0-preview1
+* SVN 版本号早于 r61242 的全部版本
+
+## 致谢
+
+感谢 Heroku 产品安全团队的 Etienne Stalmans 报告了这一问题。
+
+## 历史
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2017-12-14-net-ftp-command-injection-cve-2017-17405.md]: Missing a blank line below.

**Comment by dsh0416** [comment]
Problems mentioned have been fixed.

**Comment by JuanitoFatas** [comment]
Thank you @dsh0416 👍 !!!

**Pull request merged**